taskvine/src/manager/vine_manager.c

```diff
@@ -316,8 +316,9 @@ static int handle_cache_update( struct vine_manager *q, struct vine_worker_info
 		remote_info->in_cache = 1;
 		
 		vine_current_transfers_remove(q, id);
-
-		vine_txn_log_write_cache_update(q,w,size,transfer_time,cachename);
+		
+	   /* Do not write a cache-update to the log without an ID . such as a mini-task or buffer */
+	   if(strcmp(id, "X") != 0)vine_txn_log_write_cache_update(q,w,size,transfer_time,cachename, id);
```

Comment:
This doesn't look right: we **do** want cache-update messages for mini-tasks and other things in the transaction log, at the very least so we can visualize when the worker is taking these actions.  (See Barry's latest viz graph as an example.)